<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>424</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>558</width>
    <height>424</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AppImage Updater</string>
  </property>
  <property name="windowIcon">
   <iconset resource="app_resources/resources.qrc">
    <normaloff>:/logo.png</normaloff>:/logo.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="mainGridLayout">
     <item row="0" column="0">
      <widget class="QStackedWidget" name="mainStackedWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="firstPage">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="5" column="2">
          <widget class="QLabel" name="dragAndDropArea">
           <property name="maximumSize">
            <size>
             <width>340</width>
             <height>280</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="app_resources/resources.qrc">:/dotted_square.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="7" column="0" colspan="5">
          <widget class="QLabel" name="instruction">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <italic>false</italic>
            </font>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Drag 'n' Drop AppImage(s) to Update!</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="exitBtn">
           <property name="text">
            <string>Exit</string>
           </property>
           <property name="icon">
            <iconset resource="app_resources/resources.qrc">
             <normaloff>:/exit.png</normaloff>:/exit.png</iconset>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="aboutBtn">
           <property name="text">
            <string>About</string>
           </property>
           <property name="icon">
            <iconset resource="app_resources/resources.qrc">
             <normaloff>:/about.png</normaloff>:/about.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="loadingPage">
        <layout class="QGridLayout" name="gridLayout">
         <item row="3" column="0">
          <widget class="QLabel" name="loaderGif">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>150</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>150</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="app_resources/resources.qrc">:/loader.gif</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="inStatus">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <italic>true</italic>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="updatingPage">
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="currentUpdatingAppImage">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <italic>true</italic>
            </font>
           </property>
           <property name="midLineWidth">
            <number>0</number>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="4" column="0" colspan="2">
          <widget class="QLabel" name="updateTransferSpeed">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QProgressBar" name="updateProgressBar">
           <property name="value">
            <number>24</number>
           </property>
          </widget>
         </item>
         <item row="5" column="0" colspan="2">
          <widget class="QPlainTextEdit" name="loggerDisplayBox">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="acceptDrops">
            <bool>true</bool>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAsNeeded</enum>
           </property>
           <property name="tabChangesFocus">
            <bool>false</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="plainText">
            <string/>
           </property>
           <property name="overwriteMode">
            <bool>false</bool>
           </property>
           <property name="backgroundVisible">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QPushButton" name="CancelUpdateBtn">
           <property name="text">
            <string>Cancel Update</string>
           </property>
           <property name="icon">
            <iconset resource="app_resources/resources.qrc">
             <normaloff>:/exit.png</normaloff>:/exit.png</iconset>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QPushButton" name="ShowLogMessagesBtn">
           <property name="text">
            <string>More Information</string>
           </property>
           <property name="icon">
            <iconset resource="app_resources/resources.qrc">
             <normaloff>:/more_info_icon.png</normaloff>:/more_info_icon.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="finalPage">
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="1">
          <widget class="QLabel" name="resultIcon">
           <property name="minimumSize">
            <size>
             <width>212</width>
             <height>212</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>212</width>
             <height>212</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="2" colspan="4">
          <widget class="QLabel" name="resultLbl">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
             <italic>true</italic>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="5">
          <widget class="QPushButton" name="backToFirstPage">
           <property name="text">
            <string>Back</string>
           </property>
           <property name="icon">
            <iconset resource="app_resources/resources.qrc">
             <normaloff>:/back.png</normaloff>:/back.png</iconset>
           </property>
          </widget>
         </item>
         <item row="2" column="1" colspan="2">
          <widget class="QPushButton" name="openNewAppImage">
           <property name="text">
            <string>Open New AppImage</string>
           </property>
           <property name="icon">
            <iconset resource="app_resources/resources.qrc">
             <normaloff>:/open_appimage_icon.png</normaloff>:/open_appimage_icon.png</iconset>
           </property>
          </widget>
         </item>
         <item row="2" column="3" colspan="2">
          <widget class="QPushButton" name="exitBtn2">
           <property name="text">
            <string>Exit</string>
           </property>
           <property name="icon">
            <iconset resource="app_resources/resources.qrc">
             <normaloff>:/exit.png</normaloff>:/exit.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="app_resources/resources.qrc"/>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
